src/cpp/ext/filters/census/context.cc

```diff
@@ -18,6 +18,7 @@
 
 #include <grpc/support/port_platform.h>
 
+#include "opencensus/trace/propagation/grpc_trace_bin.h"
```

Comment:
This should be at line 24.

https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes